releasenotes-builder/src/main/java/com/github/checkstyle/github/NotesBuilder.java

```diff
@@ -129,7 +129,9 @@ public static Result buildResult(String localRepoPath, String authToken, String
      * @param commit commit information.
      * @param commitMessage commit message.
      * @throws IOException if an I/O error occurs.
-     * @throws GitAPIException if an error occurs when accessing Git API.
```

Comment:
Please remove this in minor commit, since this is not related to changes here.

Reply:
Commit mentions Javadoc problems too, this falls under `Javadoc declaration problems inspection`

Reply:
```
Reports Javadoc comments and tags with the following problems:

- invalid tag names
- incomplete tag descriptions
- duplicated tags
- missing Javadoc descriptions

```

Reply:
Thanks for the explanation :)